lib/Chat/Parser/SystemMessage.php

```diff
@@ -514,6 +514,26 @@ public function parseMessage(Message $chatMessage): void {
 			if ($currentUserIsActor) {
 				$parsedMessage = $this->l->t('You ended the poll {poll}');
 			}
+		} elseif ($message === 'recording_started') {
+			$parsedMessage = $this->l->t('{actor} started a video recording');
+			if ($currentUserIsActor) {
+				$parsedMessage = $this->l->t('You started a video recording');
+			}
+		} elseif ($message === 'recording_stopped') {
+			$parsedMessage = $this->l->t('{actor} stopped a video recording');
```

Comment:
```suggestion
			$parsedMessage = $this->l->t('{actor} stopped the video recording');
```

Should be "the" as there can only be one?